tools/reflection_generator/java_method.py

```diff
@@ -415,11 +416,12 @@ def FormatSingleParam(self, param_type, param_name, param_string_type):
       # the way bridge uses as the condition for whether call super or
       # call wrapper in override call
       #   XWalkViewInternal view => (view instanceof XWalkViewBridge)
-      if (is_internal_class and
-          not java_data.HasInstanceCreateInternallyAnnotation()):
-        return'(%s instanceof %s)' % (param_name, java_data.GetBridgeName())
-      else:
-        return None
+      if is_internal_class:
+        if not java_data.HasInstanceCreateInternallyAnnotation():
+          return'(%s instanceof %s)' % (param_name, java_data.GetBridgeName())
+        else:
+          self._parameter_is_internal = True
+      return None
```

Comment:
It has no relation with `CustomViewCallbackHandlerInternal&CustomViewCallbackInternal`.
Its for `onShowCustomView(View view, CustomViewCallbackInternal callback)` in `XWalkUIClientInternal.java`, please notice the first parameter, its not `XWalkView`.
From the comment `XWalkViewInternal view => (view instanceof XWalkViewBridge)`, it just checks the view for XWalkView, so need to modify code for `View`.